lib/sass_spec/engine_adapter.rb

```diff
@@ -66,6 +51,10 @@ def compile(sass_filename, precision)
       [result[:stdout], result[:stderr], result[:status].exitstatus]
     end
   end
+
+  def to_s
+    "#@description #@version"
```

Comment:
Consider avoiding shorthand interpolation syntax throughout.

Search and replace from `@description` to `description` (e.g. changing an instance variable to a method) breaks these in a way that doesn't raise an exception.

This even caused outages, e.g. http://status.cloudamqp.com/incidents/vj62pnp62tj9
There is a detailed discussion here: https://bugs.ruby-lang.org/issues/10541

Reply:
Done.